tools/topology/m4/bytecontrol.m4

```diff
@@ -59,4 +66,31 @@ define(`C_CONTROLBYTES',
 `	]'
 `}')
 
+dnl C_CONTROLBYTES_READONLY(name, index, ops, base, num_regs, mask, max, tlv, priv)
```

Comment:
ditto, just a small inline comment here (I know some of the others dont have them, but that's something we need to update fro readability)

Reply:
Done